meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/resin-supervisor.service

```diff
@@ -16,11 +16,7 @@ After=\
     etc-resin\x2dsupervisor.mount
 
 [Service]
-Type=simple
-Restart=always
-RestartSec=10s
-EnvironmentFile=/etc/resin-supervisor/supervisor.conf
-EnvironmentFile=-/tmp/update-supervisor.conf
```

Comment:
This removed environment files, what were they used for, and is it okay to remove them, really? @pcarranzav

Reply:
These are now handled in the script itself and this is because now we manage the container with docker and not with systemd anymore when using update supervisor mechanism.

Reply:
Okay, I see it now, thanks!

Reply:
Looks like they're now sourced directly by start-resin-supervisor, so we still have'em
(They define SUPERVISOR_IMAGE and SUPERVISOR_TAG, and maybe a few other things like that)